src/FactCheck.jl

```diff
@@ -98,34 +98,34 @@ function Base.show(io::IO, f::Failure)
             # Fancy helper fact
             fcFunc = _factcheck_function(args[2])
             if haskey(FACTCHECK_FUN_NAMES, fcFunc)
-                println(io, replace_lf(string("\tExpected: ", sprint(show, f.lhs), " ", FACTCHECK_FUN_NAMES[fcFunc], " ", sprint(show, f.rhs))))
+                print(io, replace_lf(string("<:LF:>  Expected: ", sprint(show, f.lhs), " ", FACTCHECK_FUN_NAMES[fcFunc], " ", sprint(show, f.rhs))))
             else
-                println(io, replace_lf(string("\tExpected: ", sprint(show, f.lhs), " --> ", fcFunc, "(", sprint(show, f.rhs), ")")))
+                print(io, replace_lf(string("<:LF:>  Expected: ", sprint(show, f.lhs), " --> ", fcFunc, "(", sprint(show, f.rhs), ")")))
             end
         else
             # Normal equality-test-style fact
-            println(io, replace_lf(string("\tExpected: ", sprint(show, f.rhs), "\n", "\tOccurred: ", sprint(show, f.lhs))))
+            print(io, replace_lf(string("<:LF:>  Expected: ", sprint(show, f.rhs), "<:LF:>  Occurred: ", sprint(show, f.lhs))))
         end
     else
         error("Unknown fact type: ", f.fact_type)
     end
-    println(io, "\n<COMPLETEDIN::>")
+    println(io, "\n\n<COMPLETEDIN::>")
 end
 
 function Base.show(io::IO, e::Error)
-    println(io, "\n<IT::>", e.meta.msg != nothing ? "$(e.meta.msg)" : format_fact(e.expr))
+    println(io, "\n<IT::>", replace_lf(e.meta.msg != nothing ? "$(e.meta.msg)" : format_fact(e.expr)))
     println(io, "\n<ERROR::>Test Errored")
-    println(io, replace_lf(sprint(showerror, e.err, e.backtrace)))
+    println(io, "\n<LOG::Stack trace>", replace_lf(sprint(showerror, e.err, e.backtrace)))
```

Comment:
Can you change the error to look something like this?

![image](https://user-images.githubusercontent.com/639336/223522260-7c96d6bb-a095-434b-a55b-b3606a5341a1.png)

```text
<ERROR::>BoundsError: attempt to access 3-element Vector{Int64} at index [4]

<LOG::-Stacktrace>  [1] getindex<:LF:>    @ .\\array.jl:805 [inlined]<:LF:>  [2] #53<:LF:>
```

Reply:
I did not manage to format exceptions exactly like you wanted. I managed to put an error message in an `<ERROR::>`, but I did not find a way how to avoid repeating the message in the collapsed panel. I think it's acceptable and I guess similar to what `ex.printStackTrace()` does in Java?

Reply:
Yeah, that's fine.